libethereum/EthereumCapability.cpp

```diff
@@ -838,12 +842,13 @@ bool EthereumCapability::interpretCapabilityPacket(
     }
     catch (Exception const&)
     {
-        cnetlog << "Peer causing an Exception: "
-                << boost::current_exception_diagnostic_information() << " " << _r;
+        LOG(m_loggerError) << "Peer " << _peerID << " causing an exception: "
```

Comment:
I would use ERROR channel only for critical unrecoverable errors, this I think should better be warning at most.